doc/ref/plotting_options/interactivity.ipynb

```diff
@@ -0,0 +1,297 @@
+    "Controls the contents and layout of the hover tooltip using a list of tooltips. Each tooltip is either:\n",
+    "- A 2-tuple like (\"label\", \"@<column_name>\"), or\n",
+    "- A string representing a Bokeh field (e.g., \"@<column_name>\")\n",
+    "\n",
+    "If set, it overrides the default auto-generated tooltips.\n",
+    "\n",
+    "::: {tip} \n",
+    "Be sure to use @-prefixed column names inside each tooltip.\n",
+    ":::\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "df7a80d9-db4c-4353-9b61-2a274b9fbe04",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "opts = dict(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    hover_cols='body_mass_g', # add additional hover column\n",
+    "    frame_width=250,\n",
+    ")\n",
+    "\n",
+    "plot1 = df.hvplot.scatter(\n",
+    "    hover_tooltips=[\n",
+    "        ('Length (mm)', '@bill_length_mm'),\n",
+    "        ('Depth (mm)', '@bill_depth_mm'),\n",
+    "        ('Mass (g)', '@body_mass_g')\n",
+    "    ],\n",
+    "    title='Tooltips as list of tuples',\n",
+    "    **opts\n",
+    ")\n",
+    "\n",
+    "plot2 = df.hvplot.scatter(\n",
+    "    hover_tooltips=[\n",
+    "        '@bill_length_mm',\n",
+    "        '@body_mass_g',\n",
+    "    ], # show only these 2 columns in the tooltip\n",
+    "    title='Tooltips as list of strings',\n",
+    "    **opts\n",
+    ")\n",
+    "\n",
+    "plot1 + plot2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2a1fc016-b6bd-4ba9-bbbd-3a06660c93a8",
+   "metadata": {},
+   "source": [
+    "(options-tools)=\n",
+    "## `tools`\n",
+    "A list of interactive tools to enable on the plot. These are Bokeh tool names or tool instances (e.g., `tap`, `box_select`, or `HoverTool()`). \n",
+    "\n",
+    "Tools typically appear as a vertical toolbar on the right-hand side of the plot. Common options include:\n",
+    "- `hover` – show tooltips\n",
+    "- `tap` – click to select points\n",
+    "- `box_select`, `lasso_select` – select a region\n",
+    "- `wheel_zoom`, `pan`, `reset` – for navigation\n",
+    "\n",
+    "The `hover` tool is automatically added by default when not already in `tools`.\n",
+    "\n",
+    "If you want precise control over tooltip layout or behavior, explicitly include custom tools here and pair with `hover_tooltips`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "87308d09-a853-452f-aa34-4e198e03bf39",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "df.hvplot.scatter(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    hover_cols=['species', 'body_mass_g'],\n",
```

Comment:
Does this example need to have the `hover_xxx` arguments? I found it a little confusing.